ios/PhylloConnectModule.swift

```diff
@@ -57,22 +58,78 @@ public class PhylloConnectModule: RCTEventEmitter {
     }
     
 
-    func getEnvironment(env:String) -> PhylloEnvironment {
-        switch env {
-        case "development":
-            return PhylloEnvironment.dev
-        case "sandbox":
-            return PhylloEnvironment.sandbox
-        case "production":
-            return PhylloEnvironment.prod
-        default:
-            return PhylloEnvironment.sandbox
-        }
-    }
+
+//  @objc(getPhylloEnvironmentUrl:)
+// func getPhylloEnvironmentUrl(_ env:String) -> String {
+//   var baseUrl = ""
+//     switch env {
+//         case "development":
+//             baseUrl =  PhylloEnvironment.dev.rawValue
+//         case "sandbox":
+//             baseUrl =  PhylloEnvironment.sandbox.rawValue
+//         case "staging":
+//             baseUrl =  PhylloEnvironment.staging.rawValue
+//         case "production":
+//             baseUrl =  PhylloEnvironment.prod.rawValue
+//         default:
+//             baseUrl =  PhylloEnvironment.sandbox.rawValue
+//         }
+//       return baseUrl
+//  }
+
+ 
+//  @objc func environmentUrl() -> [String: Any]! {
+//    return ["baseUrl": "Test"]
+//  }
+@objc func getPhylloEnvironmentUrl(_ environment:String ,resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
+     let baseUrl = getEnvironment(env:environment).rawValue
+     resolve(baseUrl)
+   }
+
+func getEnvironment(env:String) -> PhylloEnvironment {
+    switch env {
+      case "development":
+        return PhylloEnvironment.dev
+      case "sandbox":
+        return PhylloEnvironment.sandbox
+      case "staging":
+        return PhylloEnvironment.staging
+      case "production":
+        return PhylloEnvironment.prod
+      default:
+        return PhylloEnvironment.sandbox
+      }
+}
+
+@objc public override static func requiresMainQueueSetup() -> Bool {
+      return true
+}
     
-    @objc public override static func requiresMainQueueSetup() -> Bool {
-        return false
-    }
+// @objc public func resolvePromise(
+//     _ resolve: RCTPromiseResolveBlock,
```

Comment:
Delete the commented code (unless this is temporary removal and we will need the same code again in future)

Reply:
yes it might be needed in future , react-native sometime hard to find bridge method , we had done try-error so with functions so that i had commented .